org.tech.JTDX.yml

```diff
@@ -0,0 +1,59 @@
+app-id: org.tech.JTDX
+runtime: org.kde.Platform
+runtime-version: 5.15-22.08
+sdk: org.kde.Sdk
+command: jtdx
+rename-icon: jtdx
+
+finish-args:
+  - --share=ipc
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --device=all
+  - --socket=pulseaudio
+  - --share=network
+
+cleanup:
+  - /man
+  - /include
+  - /share/man
+  - /share/doc
+  - /doc
+  - '*.a'
+  - '*.la'
+
+modules:
+  - shared-modules/libusb/libusb.json
+  - shared-modules/linux-audio/fftw3f.json
```

Comment:
you need to add the git submodule.

It's documented there https://github.com/flathub/shared-modules/